js/app.js

```diff
@@ -234,8 +241,20 @@ var showPage = async function(pageId) {
                                 //modifyTask(uid, originalIBT[oi], {order: ni});
                             }
                         });
+                        setTimeout(function() {
+                            if (!isTaskActive) showPage(currentPage)
+                        }, 100);
```

Comment:
There's gotta be a better way than using a bunch of timeouts, I don't like it in my gut because having hardcoded 100ms constants all over the place just doesn't  feel right.